catkin_ws/src/kpr_arm_control/src/py_arm_control/arm_control.py

```diff
@@ -68,36 +72,41 @@ def createStateMachine():
 							   transitions={'StemCutted':'MoveToDropoff',
 											'CutterOpenError':'ERROR',
 											'CutterCloseError':'Release'},
-							   remapping={	'cutterStatus':'status'})
+							   remapping={	'cutterStatus':'status',
+											'result':'result'})
 											
 		smach.StateMachine.add('MoveToDropoff', state.MoveToDropoff(moveArmTo), 
 							   transitions={'MoveOK':'Release',
-											'MoveError':'MOVE_ERR'},
-							   remapping={	'data':'request'})
+											'MoveError':'ERROR'},
+							   remapping={	'data':'request',
+											'result':'result'})
 											
 		smach.StateMachine.add('Release', state.Release(writeWithDigitalFeedback), 
 							   transitions={'ReleasedAll':'OK',
 											'GripperError':'GRAB_ERR',
 											'VacuumError':'VACU_ERR',
 											'CutterError':'CUTT_ERR',
 											'ReleaseError':'ERROR'},
-							   remapping={	'systemStatus':'status'})
+							   remapping={'systemStatus':'status',
+											'result':'result'})
 	return sm
 
 def moveArmTo(pose_target):
 	'''
 	Plan the movement to the specified position and move the arm.
 
 	@param pose_target: The Pose of the goal posisiton
-	@return True when the move succeded, False otherwise
+	@return Tuple (a, b) 
```

Comment:
I figured it would not make very much difference as python allows you to return a tuple. I found this way more descriptive for the result than the values 0, 1 and 2. But if it really bothers you I can change it.